scripts/run_integration_tests.sh

```diff
@@ -10,7 +10,7 @@ set -e
 
 # These values need to be in sync with serverless.yml, where there needs to be a function
 # defined for every handler_runtime combination
-LAMBDA_HANDLERS=("async-metrics" "sync-metrics" "http-requests" "http-error")
```

Comment:
why is this deleted?

Reply:
I added it originally, after some time I think it's was a mistake my reasoning is the following.
1. It adds an external dependency that has failed before
2. It doesn't provide a lot of value and more overhead to the suite